packages/block-editor/src/components/list-view/style.scss

```diff
@@ -488,9 +480,9 @@ $block-navigation-max-indent: 8;
 	.block-editor-list-view-leaf[aria-level="#{ $i + 1 }"]
 	.block-editor-list-view__expander {
 		@if $i - 1 >= 0 {
-			margin-left: ($icon-size * $i) + 4 * ($i - 1);
+			margin-left: (20px * $i);
```

Comment:
If we use `24` here, and on line 474—and `6px` on line 280, we have bit better vertical alignment

![CleanShot 2024-04-15 at 13 31 25](https://github.com/WordPress/gutenberg/assets/1813435/214f7422-8c15-4410-8ec9-401aea224aa9)

![CleanShot 2024-04-15 at 13 30 51](https://github.com/WordPress/gutenberg/assets/1813435/19936570-8f7a-4bf5-9c27-13576bfb65ff)

Reply:
But I still wonder if it's just a bit too tight vertically, with the icon sizes we currently have. Maybe it's fine?


| Before  | After |
| ------------- | ------------- |
|![CleanShot 2024-04-15 at 13 33 55](https://github.com/WordPress/gutenberg/assets/1813435/56d150a2-87e6-4724-8419-903991b5b35f)|![CleanShot 2024-04-15 at 13 34 43](https://github.com/WordPress/gutenberg/assets/1813435/7966853a-6827-4399-9bf8-cf474cc67587)|

Reply:
I feel like it's fine. We _could_ add a 4px gap between the 32px items, but I wonder if that would be a worse experience in practice, since you could click between items and not select anything?

Reply:
The motivation here is to fit more blocks into the container before scrolling occurs to make navigating complex documents easier, so I don't think we should _increase_ the vertical spacing.

I'm not a fan of using values outside of the 4px baseline unless absolutely necessary, so would prefer to stick with 4 or 8px for the space between icon + label. I still think 4px is fine, personally, given the motivations outlined above. But will leave it at 8px if there's a strong feeling about that.

I like the other suggestions 👍

Reply:
> with the icon sizes we currently have

Not for this PR, but fwiw I wouldn't be opposed to trying _slightly_ smaller icons here. 

<img width="350" alt="Screenshot 2024-04-16 at 14 52 07" src="https://github.com/WordPress/gutenberg/assets/846565/b70165ea-38ac-470c-8d91-8f50d953622a">

Reply:
I wouldn't mind that, but I don't think the icons scale sharply.